nextflow_schema.json

```diff
@@ -109,26 +110,37 @@
                 },
                 "arriba_ref_blacklist": {
                     "type": "string",
+                    "format": "file-path",
+                    "pattern": "\\S+\\.tsv\\.gz$",
                     "fa_icon": "far fa-file-code",
                     "description": "Path to arriba reference blacklist"
                 },
                 "arriba_ref_cytobands": {
                     "type": "string",
+                    "format": "file-path",
+                    "pattern": "\\S+\\.tsv$",
                     "fa_icon": "far fa-file-code",
                     "description": "Path to arriba reference cytobands"
                 },
                 "arriba_ref_known_fusions": {
                     "type": "string",
+                    "format": "file-path",
+                    "pattern": "\\S+\\.tsv\\.gz$",
                     "fa_icon": "far fa-file-code",
                     "description": "Path to arriba reference known fusions"
                 },
                 "arriba_ref_protein_domains": {
                     "type": "string",
+                    "format": "file-path",
+                    "pattern": "\\S+\\.gff3$",
                     "fa_icon": "far fa-file-code",
                     "description": "Path to arriba reference protein domain"
                 },
                 "arriba_fusions": {
                     "type": "string",
+                    "format": "file-path",
+                    "exists": true,
```

Comment:
here, does it mean:
i) if the argument is provided, the file must exist or
ii) the argument should always be provided?

Reply:
Number 1 is correct here :)